tests/altair_theme_test.py

```diff
@@ -0,0 +1,276 @@
+from __future__ import annotations
+
+import json
+from pathlib import Path
+
+import jinja2
+import polars as pl
+
+import altair as alt
+from vega_datasets import data
+
+common_data = pl.DataFrame(
+    [
+        {"Index": 1, "Value": 28, "Position": 1, "Category": "A"},
+        {"Index": 2, "Value": 55, "Position": 2, "Category": "A"},
+        {"Index": 3, "Value": 43, "Position": 3, "Category": "A"},
+        {"Index": 4, "Value": 91, "Position": 4, "Category": "A"},
+        {"Index": 5, "Value": 81, "Position": 5, "Category": "A"},
```

Comment:
### TODO
- Rewrite using either a `dict` or some `SchemaBase` wrapper
	- No real need for adding a `polars` dependency in here